models/articles.model.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (article_id) => {
+    return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id]).then((result)=>{
```

Comment:
Great use of a parameterized query for article_id to protect against SQL injection.